Gemfile

```diff
@@ -75,12 +75,12 @@ end
 group :test, :development do
   gem 'bullet', '~> 5.0.0'
   gem 'factory_girl_rails', '~> 4.7.0'
-  gem 'pry', '~> 0.10.3'
   gem 'rspec-activemodel-mocks', '~> 1.0.1'
   gem 'rspec-rails', '~> 3.4.0'
   gem 'test-unit', '~> 3.1.0'
   gem 'spork-rails', '~> 4.0.0'
-  gem 'pry-nav', '~> 0.2.4'
+  gem 'pry-debugger', '~> 0.2.3', :platforms => :ruby_19
+  gem 'pry-byebug',  '~> 3.3.0', :platforms => [:ruby_20, :ruby_21]
```

Comment:
The version of bundler that we use (in production) is too old to recognise `:ruby_21` as a platform:

```sh
vagrant@vagrant-ubuntu-precise-64: ~/alaveteli
$ bundle
`ruby_21` is not a valid platform. The available options are: [:ruby, :ruby_18, :ruby_19, :ruby_20, :mri, :mri_18, :mri_19,
:mri_20, :rbx, :jruby, :mswin, :mingw, :mingw_18, :mingw_19, :mingw_20]
vagrant@vagrant-ubuntu-precise-64: ~/alaveteli
$ bundle --version
Bundler version 1.3.5
```